llvm/utils/update_mc_test_checks.py

```diff
@@ -118,6 +119,19 @@ def main():
         default=None,
         help="Set a default -march for when neither triple nor arch are found in a RUN line",
     )
+    parser.add_argument(
+        "--remove-duplicate",
+        action="store_true",
+        default=False,
+        help="remove duplicated test line if found",
+    )
+    parser.add_argument(
+        "--sort",
+        action="store_true",
+        default=False,
```

Comment:
I guess not having this enabled by default might be problematic. Imagine some test cases added without sorting at random positions in one patch and then sorting them in another patch as some more test cases are added. This would also move cases added in the first patch. (I assume the idea is that ultimately we want keep all MC test files sorted at all times.)

Reply:
The same reason as above. I think it's better asking user to run --sort manually.

I think we can add a hint by checking the auto-generated header line
`# auto updated by update_mc_test_check.py UTC_ARGS: --sort`

if there is a `--sort` there but there is no `--sort` being passed by the arguments, we can shot a warning